lib/order.rb

```diff
@@ -0,0 +1,46 @@
+require_relative 'menu'
+require_relative 'customer'
+include Menu
+include Customer
+
```

Comment:
By including menu and customer modules in your Order class, you have put everything into one class, thus removing almost all OOP from your program. Menu and Customer should be separate classes.